go.mod

```diff
@@ -0,0 +1,49 @@
+module github.com/WilliamMortlMicrosoft/azure-sdk-for-go-samples
```

Comment:
Module name needs to be fixed.  You might also just omit the go.mod/go.sum files for now as we don't have great module support at this time.

Reply:
Thanks for the review @jhendrixMSFT !! I made your suggested changes and added a README file